frontend/src/metabase/css/components/modal.css

```diff
@@ -10,6 +10,10 @@
 .Modal.Modal--medium  { width: 65%; }
 .Modal.Modal--wide    { width: 85%; }
 
+.Modal.Modal--tall {
```

Comment:
Is the min height here necessary? If it is that to me is a sign something funky is going on. In theory enough content in the modal would make the modal large enough and if anything you'd need a max-height.  If we can avoid it I'd prefer to not keep adding too many modifier classes on modals (see width up above).